crates/rust-client/src/lib.rs

```diff
@@ -368,3 +368,28 @@ impl FeltRng for ClientRng {
         self.0.draw_word()
     }
 }
+
+/// Represents if the client is in Debug Mode.
```

Comment:
nit: I would rewrite this to something like
```suggestion
/// Indicates whether the client is operating in debug mode.
```